db/migrate/20220525122759_init_schema.rb

```diff
@@ -0,0 +1,235 @@
+class InitSchema < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension 'pgcrypto'
+
+    create_table "add_ons", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+      t.uuid "organization_id", null: false
+      t.string "name", null: false
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)